CMakeLists.txt

```diff
@@ -0,0 +1,60 @@
+cmake_minimum_required (VERSION 2.6)
+project(ale)
+
+option(USE_SDL "Use SDL" OFF)
+option(USE_RLGLUE "Use RL-Glue" OFF)
+
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -Wno-multichar -Wunused -fno-rtti -fPIC -O3 -fomit-frame-pointer")
```

Comment:
Do we need no-rtti? The internet suggests this is only useful to save a bit of space, and it kills dynamic_cast, which on the long run could cause someone headaches.

Also (and somewhat pedantically), why -Wno-multichar?